pages/common/minikube-start.md

```diff
@@ -0,0 +1,24 @@
+# minikube start
+
+> Start minikube with different configurations.
+> More information: <https://minikube.sigs.k8s.io/docs/>.
```

Comment:
```suggestion
> More information: <https://minikube.sigs.k8s.io/docs/commands/start/>.
```
Since `minikube start` has its own docs page, we can link to it directly.